app/components/dashboard_component.rb

```diff
@@ -63,6 +63,8 @@ def include_job_applications?
   end
 
   def selected_scope
+    return "awaiting_feedback_recently_expired" if @selected_type == "awaiting_feedback"
+
     @selected_type == "published" ? "live" : selected_type
```

Comment:
Not sure a guard clause is the best tool when conditionally operating over possible values of  `@selected_type`.

IMO `case` statement fits better?
```suggestion
    case @selected_type
    when "awaiting_feedback" then "awaiting_feedback_recently_expired"
    when "published" then "live"
    else selected_type
    end
```

Reply:
Yeah, that looks much better, thanks!